contributing.md

```diff
@@ -0,0 +1,56 @@
+# Rootstock Contribution Guidelines - RIF Relay
+Thank you for investing your time in contributing to RIF Relay! Any contribution you make will be reflected on the [RIF Relay GitHub page](https://github.com/rsksmart/rif-relay). Your efforts help improve the Rootstock and RIF ecosystem.
+
+## Understanding the Project
+RIF Relay is a secure transaction relay system that enables users to pay fees using ERC-20 tokens; it was inspired by the Gas Station Network (GSN) project. It comprises four main modules: RIF Relay Contracts, RIF Relay Common, RIF Relay Client, and RIF Relay Server, each with specific functionalities and roles within the system.
```

Comment:
RIF Relay common has been removed. Furthermore, I'd also add the link to their respective repositories.
```suggestion
RIF Relay is a secure transaction relay system that enables users to pay fees using ERC-20 tokens; it was inspired by the Gas Station Network (GSN) project. It comprises four main modules: RIF Relay Contracts, RIF Relay Client, and RIF Relay Server, each with specific functionalities and roles within the system.
```